packages/new-polymath-ui/src/components/inputs/Checkbox/Checkbox.tsx

```diff
@@ -58,4 +58,16 @@ export const CheckboxPrimitive: FC<Props> = ({
   );
 };
 
-export const Checkbox = formikProxy(CheckboxPrimitive);
+const EnhancedCheckbox: FC<FormikExternalProps> = ({
+  field,
+  form,
+  ...rest
+}) => (
+  <FormikProxy<Value>
```

Comment:
`Value` isn't defined anywhere

Reply:
interesting, how did this not get signaled by the editor